packages/now-node/src/dev-server.ts

```diff
@@ -1,3 +1,26 @@
+import { register } from 'ts-node';
+
+// Use the project's version of TypeScript if available,
+// otherwise fall back to using the copy that `@now/node` uses.
+let compiler: string;
+try {
+  compiler = require.resolve('typescript', {
+    paths: [process.cwd(), __dirname],
+  });
+} catch (e) {
+  compiler = 'typescript';
+}
+
+register({
+  compiler,
+  compilerOptions: {
+    allowJs: true,
+    esModuleInterop: true,
+    jsx: 'react',
```

Comment:
Will the user's `tsconfig.json` override these settings? I wonder if someone would want `jsx: preserve`?